.github/workflows/ovn-kubernetes.yml

```diff
@@ -0,0 +1,170 @@
+name: ovn-kubernetes
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+  # Run Sunday at midnight
+  - cron: '0 0 * * 0'
+
+env:
+  GO_VERSION: "1.16.3"
+  K8S_VERSION: v1.20.2
+  OVNKUBE_COMMIT: "f669ef6d35c86bf46d7a1055072e48ea1506f88c"
+  KIND_CLUSTER_NAME: ovn
+  KIND_INSTALL_INGRESS: true
+  KIND_ALLOW_SYSTEM_WRITES: true
+  # This skips tests tagged as Serial
+  # Current Serial tests are not relevant for OVN
+  PARALLEL: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out ovn
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+
+    - name: Build ovn-kubernetes container
+      run: |
+        docker build --build-arg OVNKUBE_COMMIT=${{ env.OVNKUBE_COMMIT }} -t ovn-daemonset-f:dev -f .ci/ovn-kubernetes/Dockerfile .
+        mkdir /tmp/_output
+        docker save ovn-daemonset-f:dev > /tmp/_output/image.tar
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: test-image
+        path: /tmp/_output/image.tar
+
+  e2e:
+    name: e2e
+    if: github.event_name != 'schedule'
+    runs-on: ubuntu-latest
```

Comment:
Based on the previous discussion, I guess this needs to be changed to `ubuntu-20.04`

Reply:
right, thanks for catching it, I totally forget it

Reply:
This still didn't change to `ubuntu-20.04` :)

Reply:
hmm, forget to push or what XD

Reply:
haha, indeed, I forget to push, lool